xcode/Subconscious/Shared/UIKit/BlockEditor/BlockEditorViewController.swift

```diff
@@ -259,7 +259,7 @@ extension BlockEditor {
                 for: indexPath
             ) as! TextBlockCell
             cell.delegate = self
-            cell.render(state)
+            cell.update(parentController: self, state: state)
```

Comment:
Need to pass down reference to parent controller for UIHostingView.

Dropping the `.render(state:)` idea. It's a nice signature, but isn't living up to the reality of what UIKit API is demanding of me :/